saleor/account/models.py

```diff
@@ -160,6 +160,21 @@ def get_ajax_label(self):
                 address.first_name, address.last_name, self.email)
         return self.email
 
+    def get_addresses(self):
```

Comment:
This looks like a good candidate for a custom queryset for the Address model that we could use like: `user.addresses.annotate_default(user)`.

Reply:
Sorry I didn't get it. Could you give further explanation please?

Reply:
I'd create something like this:
```
# account/models.py

class AddressQueryset(models.QuerySet):
    def annotate_default(self, user):
        # logic from `get_addresses`
        return user.addresses.annotate(...)

class Address(models.Model):
    ...
    objects = AddressQueryset.as_manager()
```

And then in the `User` type in addresses resolver you could use `return self.addresses.annotate_default(self)` (where `self` is the `User` instance).

Reply:
Got it! Yeah it's better to move to queryset manager. Will refactor it tomorrow :D